### DISCUSSION TITLE
Report on Consumption of Tokens and Charges

### DISCUSSION THREAD

**Discussion by marketcalls** [cline/cline#841] [Feature Requests]
It would be great if there is a report on Overall Consumption of Tokens and Charges.

this month i have to manually count the tokens and pricing from each and every thread 
and found out that total tokens exceeded 225 Million tokens. Any automation this tracking
the token consumption could be a better feature for heavy users.

Kind of Monthly or Weekly report sort of?

I would like to hear your thoughts on this.

**Comment by m-check1B** [comment]
Anthropic:
Determine the number of tokens in a message before sending it to Claude to make informed decisions about your prompts and usage.
